docs/port-configuration.md

```diff
@@ -18,27 +14,14 @@ defined ports not get exposed in the main service. For example, by default,
 the `mysql` section of the `values.yaml` file contains this definition:
 
 ```yaml
-mysql:
-  enabled: true
-  ...
-  ports:
-    ...
-    # ports that are added to the sidecar service
-    sidecar:
-      - 3306
-      - 3307
-      - 3308
- ...
+containerPorts:
+  mysql: 3306
+  pg: 5432
```

Comment:
It would be helpful to either add more context on the port alias ( `mysql` and `pg` in this example) OR point to the documentation in the chart that explains how they work. A good example to be added is a mongodb configuration with 3 ports (`mongodb0`, `mongodb1` and `mongodb2`) to illustrate the names could be arbitrary.